lib/blueberry_rails/app_builder.rb

```diff
@@ -165,6 +165,14 @@ def setup_capistrano
       template 'capistrano_dotenv.cap', 'lib/capistrano/tasks/dotenv.cap'
     end
 
+    def configure_simple_form
+      if options[:bootstrap]
+        generate 'simple_form:install --bootstrap'
+      else
+        generate 'simple_form:install'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.